Mathlib/Geometry/Manifold/IntegralCurve.lean

```diff
@@ -532,4 +604,170 @@ lemma IsIntegralCurve.periodic_xor_injective [BoundarylessManifold I M]
       rw [abs_of_nonneg hab]
       exact hγ.periodic_of_eq hv heq
 
+section UniformTime
+
+lemma exists_isIntegralCurveOn_Ioo_eqOn [BoundarylessManifold I M]
+    (hv : ContMDiff I I.tangent 1 (fun x => (⟨x, v x⟩ : TangentBundle I M))) {x : M}
```

Comment:
Optional aside:

I know this pattern already exists elsewhere in this file but if you have the appetite, I think we need better syntax for saying "v is Cⁿ". Possibly an `abbrev` or possibly notation (or even both).

Reply:
There's [ContMDiffSection](https://leanprover-community.github.io/mathlib4_docs/Mathlib/Geometry/Manifold/VectorBundle/SmoothSection.html#ContMDiffSection). I hesitated to use it because I wanted to preserve the symmetry between the `ContMDiffAt` assumption for local integral curves and `ContMDiff` for global integral curves. That's not a very strong reason though.

Reply:
Also from #9341, I think there's enough justification to introduce notation for tangent fields. Let's leave that for another PR.